go.mod

```diff
@@ -1,22 +1,23 @@
 module github.com/lazyledger/lazyledger-core
 
-go 1.14
+go 1.15
 
 require (
 	github.com/BurntSushi/toml v0.3.1
 	github.com/ChainSafe/go-schnorrkel v0.0.0-20200405005733-88cbf1b4c40d
 	github.com/Workiva/go-datastructures v1.0.52
 	github.com/btcsuite/btcd v0.21.0-beta
 	github.com/btcsuite/btcutil v1.0.2
-	github.com/confio/ics23/go v0.0.0-20200817220745-f173e6211efb
-	github.com/cosmos/iavl v0.15.0-rc3.0.20201009144442-230e9bdf52cd
+	github.com/confio/ics23/go v0.6.3
+	github.com/cosmos/iavl v0.15.0
```

Comment:
Wasn't iavl removed as a dependency from tendermint?

Reply:
Nope it was added back for testing. We have an open issue to solve this: https://github.com/tendermint/tendermint/issues/5694